fix(docs): update hydra-node-options section (#544)
input-output-hk/hydra: docs/docs/getting-started/quickstart.md

`--host` <br/> `--port`                                       | This Hydra node host and port, to which peers from the Hydra network can connect to.
`--peer`                                                      | The Hydra network peers address.<br />Must be provided multiple time, one for each peer.
`--hydra-verification-key` <br/> `--cardano-verification-key` | The Cardano and Hydra credentials for peers.<br />Those options must also be provided multiple times, depending on the number of peers.
`--hydra-signing-key` <br/> `--cardano-signing-key`           | The Cardano and Hydra credentials for the node itself.
`--hydra-scripts-tx-id`                                       | A transaction id referencing Hydra's Plutus scripts.<br />See release notes.
`--node-socket`                                               | The Cardano node's IPC socket filepath.<br />Used for inter-process communication with the node.
`--ledger-genesis` <br/> `--ledger-protocol-parameters`       | The Hydra ledger rules and parameters for the head.

Also, optionally:

Options                         | Description
---                             | ---

> `--verbosity`                   | Turn on/off logging. Default is on. Turn off by passing 'quiet' as value.

Reviewer: This is wrong. We only have `--quiet`
```suggestion
`--quiet`                   | Turn off logging when given, default is on.
```

`--api-host` <br/> `--api-port` | The Hydra API host and port, to interact with the [WebSocket API](/api-reference).
`--monitoring-port`             | The port this node listens on for monitoring and metrics via Prometheus.<br />If left empty, monitoring server is not started.

> `--network-id`                  | The id of a test network to conect, with the given network magic.

Reviewer: ```suggestion
`--network-id`                  | Network magic of the **testnet** to connect to.
```

> `--start-chain-from`            | The id of the block we want to start observing the ledger from.<br />Composed by the slot number, a saparor (".") and the hash of the block header.<br />(i.e.: 52970883.d36a9936ae7a07f5f4bdc9ad0b23761cb7b14f35007e54947e27a1510f897f04)

Reviewer: ```suggestion
`--start-chain-from`            | The id of the block we want to start observing the ledger from.<br />Composed by the slot number, a separator (".") and the hash of the block header. For example: 52970883.d36a9936ae7a07f5f4bdc9ad0b23761cb7b14f35007e54947e27a1510f897f04
```
